RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -23,6 +23,9 @@
 from RecoJets.JetProducers.fixedGridRhoProducerFastjet_cfi import *
 fixedGridRhoFastjetAllTmp = fixedGridRhoFastjetAll.clone(pfCandidatesTag = cms.InputTag("particleFlowTmp"))
 
+from RecoJets.JetProducers.ak4PFJets_cfi import *
+ak4PFJetsTmp = ak4PFJets.clone(src = cms.InputTag("particleFlowTmp"))
```

Comment:
how much more wrong will it get with just the calojets used in the default PV sorting?